packages/base/src/3dview/mainview.tsx

```diff
@@ -1484,11 +1473,20 @@ export class MainView extends React.Component<IProps, IStates> {
     if (this._explodedView.enabled) {
       const center = new THREE.Vector3();
       this._boundingGroup.getCenter(center);
+      
 
       this._explodedViewLinesHelperGroup?.removeFromParent();
       this._explodedViewLinesHelperGroup = new THREE.Group();
 
       for (const group of this._meshGroup?.children as THREE.Group[]) {
+        const groupMetadata = group.userData as IMeshGroupMetadata;
+        if (!groupMetadata.originalPosition) {
+          console.warn(
+            `Group ${group.name} is missing originalPosition metadata. Preventing exploded view.`
+          );
+          continue;
+        }
```

Comment:
Let's not complicate our code with this, we should assume it's there and let it fail if it's not there.